CRM/Utils/Hook.php

```diff
@@ -1805,6 +1805,28 @@ public static function triggerInfo(&$info, $tableName = NULL) {
     );
   }
 
+  /**
+   * Define the list of fields supported in APIv4 data-translation.
+   *
+   * @param array $fields
+   *   List of data fields to translate, organized by table and column.
+   *   Omitted/unlisted fields are not translated. Any listed field may be translated.
+   *   Values should be TRUE-ish.
+   *   Ex: $fields['civicrm_event']['summary'] = TRUE
+   *   Ex: $fields['civicrm_event']['summary'] = 'yesplease';
+   *
+   * At time of writing, the `$fields` list x is prepopulated based on `<localizable>` fields in core's `xml/schema`.
```

Comment:
@totten I thought we decided we would not use existing localizable as that would lead to both systems trying to do the same field (we could add 'translatable' with 1 being ignored if both is set)

Reply:
Correct. I changed the logic, but I'll update the stale comment.